### DISCUSSION TITLE
New colors: Sunset and Citron

### DISCUSSION THREAD

**Discussion by edgarasben** [tailwindlabs/tailwindcss#4138] [Ideas]
While working on some projects I felt there is some space for new colors between _Lime_ and _Yellow_ and between _Orange_ and _Red_.

My suggestion is to add 2 new colors: **Sunset** and **Citron**.

![image](https://user-images.githubusercontent.com/725324/115522362-5465a200-a28c-11eb-894b-f291e13493a1.png)

Also I have an idea to suggest adding Turquoise, but for that I think an existing _Teal_ has to be more blue. So this is still work in progress:
<img width="808" alt="image" src="https://user-images.githubusercontent.com/725324/115522784-b3c3b200-a28c-11eb-8e7d-bd548d3d283f.png">

**Comment by surjithctly** [comment]
This is so cool. 
Can you also share the hex code in `tailwind.config` format?

**Reply by edgarasben** [reply]
Sure!

```js
const colors = require('tailwindcss/colors')

module.exports = {
  theme: {
    extend: {
      colors: {
        sunset: {
          50: '#fff3f0',
          100: '#ffe3db',
          200: '#ffc7b8',
          300: '#fca790',
          400: '#f48465',
          500: '#f16741',
          600: '#de461b',
          700: '#c8360e',
          800: '#9e2605',
          900: '#811f03',
        },
        citron: {
          50: '#fdffdf',
          100: '#fbffbe',
          200: '#f6fd90',
          300: '#ebf468',
          400: '#dce832',
          500: '#cbd90d',
          600: '#afbb0b',
          700: '#868f09',
          800: '#707808',
          900: '#575e02',
        }
      },
    },
  },
  variants: {},
  plugins: [],
}
```
